### PR TITLE
Connected the fetch users api 

### PR DESCRIPTION
**Pull request by silasade**

### CODE REVIEW

File: src/Pages/Subadmin/ManageUsers/ManageUsers.tsx
Comment (a-k-o-r-e-d-e) [inline src/Pages/Subadmin/ManageUsers/ManageUsers.tsx:1]: store the base url in a constants

**Review by a-k-o-r-e-d-e — CHANGES_REQUESTED** [review]

File: src/Pages/Subadmin/ManageUsers/ManageUsers.tsx
Comment (silasade) [reply src/Pages/Subadmin/ManageUsers/ManageUsers.tsx:1]: done

File: src/Global Components/AppConstants/AppConstants.tsx
Comment (a-k-o-r-e-d-e) [inline src/Global Components/AppConstants/AppConstants.tsx:1]: 'AppConstants'

**Review by a-k-o-r-e-d-e — CHANGES_REQUESTED** [review]

File: src/Global Components/AppConstants/AppConstants.tsx
Comment (silasade) [reply src/Global Components/AppConstants/AppConstants.tsx:1]: It has been renamed.

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Review by a-k-o-r-e-d-e — APPROVED** [review]

**Pull request merged**